pyproject.toml

```diff
@@ -34,6 +34,7 @@ dynamic = [
 ]
 dependencies = [
   "pyproject-fmt-rust==1.0.4",
+  "tomli; python_version<'3.11'",
```

Comment:
Please add lower bound for this requirement.

Reply:
added